src/components/organism/PoolLiquidity/index.tsx

```diff
@@ -115,6 +123,7 @@ const PoolLiquidity = () => {
       }
     } catch (error) {
       dotAcpToast.error(`Error: ${error}`);
+      console.log(error);
```

Comment:
remove console log. if you need it, add it only during develop. When u create pr it should be without it